utils/thapi.h

```diff
@@ -0,0 +1,7 @@
+#if !defined(THAPI)
+#define THAPI
+
+void thapi_start(void);
+void thapi_stop(void) __attribute__((constructor));
```

Comment:
We  should do the trick to do the `+1` lttng-constructor. Just to be future proof (aka if on day we need to run something after the thapi_stop constructor).